grafana/dashboards.sh

```diff
@@ -57,18 +57,14 @@ backup() {
 restore() {
 	find "$DASHBOARDS_DIRECTORY" -type f -name \*.json -print0 |
 		while IFS= read -r -d '' dashboard_path; do
-			folder_id=$(get_folder_id "$(basename "$dashboard_path" .json)")
+			folder_id=$(get_folder_id "$(jq -r '.uid' <"$dashboard_path")")
 			curl \
 				--silent --fail --show-error --output /dev/null \
 				--user "$LOGIN" \
 				-X POST -H "Content-Type: application/json" \
 				-d "{\"dashboard\":$(cat "$dashboard_path"), \
                     \"overwrite\":true, \
-                    \"folderId\":$folder_id, \
-                    \"inputs\":[{\"name\":\"DS_CLOUDWATCH\", \
-                                 \"type\":\"datasource\", \
-                                 \"pluginId\":\"cloudwatch\", \
-                                 \"value\":\"TeslaMate\"}]}" \
+                    \"folderId\":$folder_id}" \
 				"$URL/api/dashboards/import"
```

Comment:
can we switch to the officially documented endpoint here ?

`POST /api/dashboards/db`

Reply:
Sure, done!